### DISCUSSION TITLE
Incomplete Input data for Create Variable

### DISCUSSION THREAD

**Discussion by JesseTee5** [postmanlabs/postman-flows#74] [Defect]
I'm running into what may be a bug when trying to create a variable from an API call. One of my endpoints is able to map specific attributes within the List>Index Object but my second endpoint just gives me an Index-Any with no contained attributes available. I've setup the Flow in multiple configurations to try to get the input data to show up properly but no luck.

I've got screenshots of both and the endpoints responses are structured in a similar way so I'm not sure why the input data isn't showing the attribute I need to map to my variable.

Screenshots included showing both:
<img width="857" alt="Screen Shot 2021-10-26 at 10 01 24 AM" src="https://user-images.githubusercontent.com/93212498/138917916-8f5d3767-3f6b-44dd-9396-176564e7fccc.png">
<img width="830" alt="Screen Shot 2021-10-26 at 10 00 43 AM" src="https://user-images.githubusercontent.com/93212498/138917925-2a3fd884-5608-49d3-87a6-6f6304a33437.png">

**Comment by saswatds** [comment]
Hey, @JesseTee5 this seems like a bug in our data type inference layer. Looking into this immediately.
